apps/workers/sql-load/worker.go

```diff
@@ -465,3 +501,94 @@ func (ds *DataSet) findInsertKey(name string) bool {
 	}
 	return false
 }
+
+func RandString(n int) string {
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
you probably could move the seed as part of an init function 
``` go
func init() {
rand.Seed(time.Now().UTC().UnixNano())
}
```

Reply:
That would probably work as well. Is there any benefit to moving it?